src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public class Circle extends Color {
```

Comment:
Circle should not extend Color. Inheritance is used to model a 'is-a' relationship, and a Circle is not a Color. Consider using composition instead, where Circle has a Color.